css-three-column-layout/styles.css

```diff
@@ -0,0 +1,82 @@
+* {
+  background-color: #1996da;
+  font-family: Lato, sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  src:
+    url("https://fonts.gstatic.com/s/lato/v15/S6uyw4BMUTPHjx4wXiWtFCc.woff2")
+    format("woff2");
+  unicode-range:
+    u+0000-00ff,
+    u+0131,
+    u+0152-0153,
+    u+02bb-02bc,
+    u+02c6,
+    u+02da,
+    u+02dc,
+    u+2000-206f,
+    u+2074,
+    u+20ac,
+    u+2122,
+    u+2191,
+    u+2193,
+    u+2212,
+    u+2215,
+    u+feff,
+    u+fffd;
```

Comment:
I didn't teach you any of these, they are unnecessary for you to use 😂

Reply:
> I didn't teach you any of these, they are unnecessary for you to use 😂

LOL Sorry! I thought we had to copy everything from the fonts.css to be able to use the font. I just took it off and re-submitted.